### ISSUE TITLE
[2024/06/05] FINE-TUNING LANGUAGE MODELS FOR FACTUALITY (ICLR 2024)

### ISSUE THREAD

**Issue by bangawayoo** [zaemyung/Seeking-SOTA#144]
- Title: [FINE-TUNING LANGUAGE MODELS FOR FACTUALITY (ICLR 2024)](https://arxiv.org/abs/2311.08401)
- Authors: Katherine Tian*†, Eric Mitchell*†, Huaxiu Yao†§, Christopher D. Manning†, Chelsea Finn†  (†Stanford University §UNC Chapel Hill)

> Abstract: The fluency and creativity of large pre-trained language models (LLMs) have led to their widespread use, sometimes even as a replacement for traditional search engines. Yet language models are prone to making convincing but factually in- accurate claims, often referred to as ‘hallucinations.’ These errors can inadver- tently spread misinformation or harmfully perpetuate misconceptions. Further, manual fact-checking of model responses is a time-consuming process, making human factuality labels expensive to acquire. In this work, we fine-tune lan- guage models to be more factual, without human labeling and targeting more open-ended generation settings than past work. We leverage two key recent in- novations in NLP to do so. First, several recent works have proposed meth- ods for judging the factuality of open-ended text by measuring consistency with an external knowledge base or simply a large model’s confidence scores. Sec- ond, the Direct Preference Optimization algorithm enables straightforward fine- tuning of language models on objectives other than supervised imitation, us- ing a preference ranking over possible model responses. We show that learn- ing from automatically generated factuality preference rankings, generated either through existing retrieval systems or our novel retrieval-free approach, signifi- cantly improves the factuality (percent of generated claims that are correct) of Llama-2 on held-out topics compared with RLHF or decoding strategies targeted at factuality. At 7B scale, compared to Llama-2-Chat, we observe 53% and 50% reduction in factual error rate when generating biographies and answer- ing medical questions, respectively.
